Libs/DICOM/Widgets/ctkDICOMAppWidget.cpp

```diff
@@ -325,7 +333,9 @@ ctkDICOMAppWidget::~ctkDICOMAppWidget()
 {
   Q_D(ctkDICOMAppWidget);
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.